package.json

```diff
@@ -28,8 +28,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "marked": "*",
-    "simplemde": "*"
+    "easymde": "https://github.com/Ionaru/easy-markdown-editor",
```

Comment:
Why is the git repository used here?

Reply:
Agreed. At that point I didn't notice that this fork has been released as a new package. Sorry for that.

Reply:
That's OK. Thanks for your work.